picky/src/x509/pkcs7/authenticode.rs

```diff
@@ -0,0 +1,1886 @@
+        self.inner.borrow_mut().strictness.require_not_before_check = false;
+        self
+    }
+
+    #[inline]
+    pub fn require_not_after_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_not_after_check = true;
+        self
+    }
+
+    #[inline]
+    pub fn ignore_not_after_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_not_after_check = false;
+        self
+    }
+
+    #[inline]
+    pub fn require_signing_certificate_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_signing_certificate_check = true;
+        self
+    }
+
+    #[inline]
+    pub fn ignore_signing_certificate_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_signing_certificate_check = false;
+        self
+    }
+
+    #[inline]
+    pub fn require_basic_authenticode_validation(&self, expected_file_hash: Vec<u8>) -> &Self {
+        self.inner.borrow_mut().strictness.require_basic_authenticode_validation = true;
+        self.inner.borrow_mut().expected_file_hash = Some(expected_file_hash);
+        self
+    }
+
+    #[inline]
+    pub fn ignore_basic_authenticode_validation(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_basic_authenticode_validation = false;
+        self.inner.borrow_mut().expected_file_hash = None;
+        self
+    }
+
+    #[cfg(feature = "ctl")]
+    #[inline]
+    pub fn require_ca_against_ctl_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_ca_verification_against_ctl = true;
+        self
+    }
+
+    #[cfg(feature = "ctl")]
+    #[inline]
+    pub fn ignore_ca_against_ctl_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_ca_verification_against_ctl = false;
+        self
+    }
+
+    pub fn require_chain_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_chain_check = true;
+        self
+    }
+
+    pub fn ignore_chain_check(&self) -> &Self {
+        self.inner.borrow_mut().strictness.require_chain_check = false;
+        self
+    }
+
+    #[cfg(feature = "ctl")]
+    #[inline]
+    pub fn exclude_cert_authorities(&self, excluded_cert_authorities: &'a [DirectoryName]) -> &Self {
+        self.inner
+            .borrow_mut()
+            .strictness
+            .exclude_specific_cert_authorities_from_ctl_check = true;
+        self.inner
+            .borrow_mut()
+            .excluded_cert_authorities
+            .extend_from_slice(excluded_cert_authorities);
+
+        self
+    }
+
+    #[cfg(feature = "ctl")]
+    #[inline]
+    pub fn ignore_excluded_cert_authorities(&self) -> &Self {
+        self.inner
+            .borrow_mut()
+            .strictness
+            .exclude_specific_cert_authorities_from_ctl_check = false;
+        self.inner.borrow_mut().excluded_cert_authorities.clear();
+
+        self
+    }
+
+    fn verify_authenticode_basic(&self) -> AuthenticodeResult<()> {
```

Comment:
Picky name convention for helpers
```suggestion
    fn h_verify_authenticode_basic(&self) -> AuthenticodeResult<()> {
```

Reply:
Applied to other helpers